mongo/integration/sessions_test.go

```diff
@@ -330,6 +330,37 @@ func TestSessions(t *testing.T) {
 	})
 }
 
+func TestSessionsStandalone(t *testing.T) {
+	standaloneOpts := mtest.NewOptions().MinServerVersion("3.6").Topologies(mtest.Single)
+	mt := mtest.New(t, standaloneOpts)
+	defer mt.Close()
+	mt.RunOpts("transaction number not sent on writes", standaloneOpts, func(mt *mtest.T) {
+		// Standalones do not support retryable writes and will error if a transaction number is sent
+
+		sess, err := mt.Client.StartSession()
+		assert.Nil(mt, err, "StartSession error: %v", err)
+		defer sess.EndSession(mtest.Background)
+
+		mt.ClearEvents()
+
+		err = mongo.WithSession(mtest.Background, sess, func(ctx mongo.SessionContext) error {
+			doc := bson.D{{"foo", 1}}
+			_, err := mt.Coll.InsertOne(ctx, doc)
+			return err
+		})
+		assert.Nil(mt, err, "InsertOne error: %v", err)
+
+		_, wantID := sess.ID().Lookup("id").Binary()
+		command := mt.GetStartedEvent().Command
+		lsid, err := command.LookupErr("lsid")
+		assert.Nil(mt, err, "Error getting lsid: %v", err)
+		_, gotID := lsid.Document().Lookup("id").Binary()
+		assert.True(mt, bytes.Equal(wantID, gotID), "expected session ID %v, got %v", wantID, gotID)
+		txnNumber := command.Lookup("txnNumber")
```

Comment:
nit: Instead of using `Validate` below, you can use `LookupErr` here and assert that there is an error.

Reply:
done